tests/util/src/opentelemetry/test/test_base.py

```diff
@@ -31,9 +30,5 @@ def setUpClass(cls):
         span_processor = export.SimpleExportSpanProcessor(cls.memory_exporter)
         cls.tracer_provider.add_span_processor(span_processor)
 
-    @classmethod
-    def tearDownClass(cls):
-        reload(trace_api)
```

Comment:
I did that because I wanted to invalidate the call to `set_tracer_provider(cls.tracer_provider)`, could it be a good idea to invalidate that by doing something like `set_tracer_provider(None)`?